unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Settings/Settings.cs

```diff
@@ -89,14 +104,34 @@ private void LoadGeneralSettings()
             }
         }
 
+        private void LoadAudioSettings() {
+            currentAudioSettings = GetDefaultAudioSettings();
+
+            if (PlayerPrefsUtils.HasKey(AUDIO_SETTINGS_KEY))
+            {
+                try
+                {
+                    object currentSetting = currentAudioSettings;
+                    JsonUtility.FromJsonOverwrite(PlayerPrefsUtils.GetString(AUDIO_SETTINGS_KEY), currentSetting);
+                    currentAudioSettings = (SettingsData.AudioSettings)currentSetting;
+                }
+                catch (Exception e)
+                {
+                    Debug.Log(e.Message);
```

Comment:
```suggestion
                    Debug.LogError(e.Message);
```

Reply:
Nice catch. I also changed the existing exceptions for QualitySettings and GeneralSettings, which follow the same pattern.